udf/extract_count_histogram_value.sql

```diff
@@ -0,0 +1,26 @@
+CREATE TEMP FUNCTION
```

Comment:
Can you add some docs here at the top of the file here? It's not obvious to me why we're extracting just the `0` key.